src/main/java/io/aiven/kafka/connect/opensearch/OpensearchSinkConnectorConfig.java

```diff
@@ -99,11 +100,16 @@ public class OpensearchSinkConnectorConfig extends AbstractConfig {
     private static final String KEY_IGNORE_DOC =
             "Whether to ignore the record key for the purpose of forming the Opensearch document ID."
                     + " When this is set to ``true``, document IDs will be generated as the record's "
-                    + "``topic+partition+offset``.\n Note that this is a global config that applies to all "
-                    + "topics, use ``" + TOPIC_KEY_IGNORE_CONFIG + "`` to override as ``true`` for specific "
+                    + "``topic+partition+offset`` or a strategy set in ``" + KEY_IGNORE_ID_STRATEGY_CONFIG + "``.\n"
+                    + "Note that this is a global config that applies to all topics, use "
+                    + "``" + TOPIC_KEY_IGNORE_CONFIG + "`` to use ID=``topic+partition+offset`` for specific "
                     + "topics.";
     private static final String TOPIC_KEY_IGNORE_DOC =
             "List of topics for which ``" + KEY_IGNORE_CONFIG + "`` should be ``true``.";
+    private static final String KEY_IGNORE_ID_STRATEGY_DOC =
+            "Specifies the strategy to generate the Document ID. Only applicable when ``" + KEY_IGNORE_CONFIG + "`` is"
+                    + " ``true``. Available strategies " + RecordConverter.DocumentIDStrategy.describe() + ".\n"
+                    + "Note that this is a global config that applies to all topics.";
```

Comment:
`Note that this is a global config that applies to all topic` - this is not strictly correct, it depends where KEY_IGNORE_DOC is set or TOPIC_KEY_IGNORE_CONFIG is set.